### DISCUSSION TITLE
Fat energy deposition in cavity  for EGS_fac

### DISCUSSION THREAD

**Discussion by BINDX93** [nrc-cnrc/EGSnrc#690] [Questions] [answered]
Could you tell me what is the meaning of fat energy deposition in cavity?  

My calculated result of EGS_fac shows several messages as below:

Fat energy deposition in cavity:
iarg=3 edep=0.00401493 wt=0.00102385 q=-1 latch=0 is_fat=1
Fat energy deposition in cavity:
  iarg=3 edep=0.00518924 wt=0.00102379 q=-1 latch=0 is_fat=1
Fat energy deposition in cavity:
  iarg=3 edep=0.00501246 wt=0.00102394 q=-1 latch=0 is_fat=1
Fat energy deposition in cavity:
  iarg=3 edep=0.00604417 wt=0.00102393 q=-1 latch=0 is_fat=1

**Comment by rtownson** [comment, marked as answer]
A fat particle results from the russian roulette variance reduction technique, but generally those particles should never reach the cavity. Are you using a phase-space that has fat particles in it, like from using DBS in BEAMnrc?

**Comment by BINDX93** [comment]
Thanks for your reply.
No, I am not. I used a point-source.
When I reduce hte number of photon splitting. this phenomenon is disappear.
I am a beginner of the EGS-fac. Thanks.
------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Reid Townson"<notifications@github.com&gt;;
发送时间:&nbsp;2019年12月13日(星期五) 凌晨2:30
收件人:&nbsp;"nrc-cnrc/EGSnrc"<EGSnrc@noreply.github.com&gt;;
抄送:&nbsp;"斌DX"<1021641320@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [nrc-cnrc/EGSnrc] Fat energy deposition in cavity  for EGS_fac (#563)




A fat particle results from the russian roulette variance reduction technique, but generally those particles should never reach the cavity. Are you using a phase-space that has fat particles in it, like from using DBS in BEAMnrc?

**Comment by rtownson** [comment]
Hi @BINDX93, you're welcome to send me your input file if this is still an unresolved issue for you.
